scRNA-seq/pre-processing_scRNA-seq.md

```diff
@@ -0,0 +1,149 @@
+#### Cons:  
+- Is not very efficient (generally 96 cells per plate)  
+- Takes much longer to run (Days/weeks depending on sample size) .
+- Is a lot more expensive.  
+
+### 2) Tag-based scRNA-seq  
+*Example:* 10X Genomics [(Zheng et al, 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601)  
+Cells are separated by emulsion/droplets, and individual cells are given barcodes.
+Everything is then sequenced.
+These types of methods, because they are newer, are more likely to have
+[Unique Molecular Identifiers (UMIs)](http://www.nature.com/doifinder/10.1038/nmeth.2772)
+which allow you to better control for PCR amplification errors and biases.
+Individual samples have two fastq files: one for the cell barcodes
+and another with the individual reads.
+
+#### Pros:  
+- Can run potentially millions of cells at once.   
+- A lot quicker computing wise.  
+- Won't take up all your computer's storage.  
+- A lot cheaper  
+
+#### Cons:  
+- Sequencing is not bidirectional so data will likely have more intense 3' bias.  
+- Coverage of these technologies generally is not as deep.  
+
+*More sources on the comparisons and explanations of these technologies: *   
+- [Zhang et al, 2018](https://doi.org/10.1016/j.molcel.2018.10.020)  
+- [AlJanahi et al, 2018](https://doi.org/10.1016/j.omtm.2018.07.003)  
+- [Angerer et al, 2017](http://dx.doi.org/10.1016/j.coisb.2017.07.004)  
+- [Baran-Gale et al, 2018](https://doi.org/10.1093/bfgp/elx035)  
+
+## Steps for Processing scRNA-seq Data:
+
+### Step 0: Obtaining the data
+10X Genomics has plenty of [datasets](https://support.10xgenomics.com/single-cell-gene-expression/datasets)
+available to play around with.
+For this example, we will use the [1k pbmc](https://support.10xgenomics.com/single-cell-gene-expression/datasets/3.0.0/pbmc_1k_v2)
+(peripheral blood mononuclear cells).
+We will process a fastq file of this as an example.  
+For practical purposes as far as time constraints, we have subset this fastq file
+and have it ready for you to process but with only the first 3 million reads.
+
+*Note*: depending on the state of the data you are working with, ie. if you have
+a `.bcl` file, you will need to use `CellRanger` with their `mkfastq` command to
+make fastq files from this.
+However, because most publicly available data is in fastq format and any
+core you might be working with will also likely provide you with fastq files,
+we are starting from this point.  
+
+### Step 1: Set up your output directory
+As described in a previous module, we'll make a directory from command line
+for us to store our output data in.
+```
+mkdir alevin_output  
+```
+
+### Step 2: Index the human transcriptome with Salmon
+Before you can quantify with Salmon and
+[Alevin](https://www.biorxiv.org/content/10.1101/335000v2), we need a transcriptome
+to be indexed.
+You can use the same trancriptome index as was used for bulk-rna-seq, however,
+due to the smaller pieces and amounts of single cell RNA-seq as opposed to bulk,
```

Comment:
It's a matter of read length if we're keeping it simple and I am inclined to keep it simple.